Assets/Resources/Microgames/KaguyaMemory/Scripts/KaguyaMemory_ItemScript.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KaguyaMemory_ItemScript : MonoBehaviour {
+
+    [SerializeField]
+    private AudioClip correctSound;
+
+    [SerializeField]
+    private AudioClip wrongSound;
+
+    public GameObject rngMaster;
+    private Vector3 startingPosition;
+    private int timer1 = 1;
+    public bool isMoving = false;
+    public bool isCorrect = false;
+    public GameObject correctIndicator;
+    public GameObject wrongIndicator;
+    public bool isClicked = false;
+
+	// Use this for initialization
+	void Start () {
+        GetComponent<SpriteRenderer>().color = new Color(1, 1, 1, 0);
+        startingPosition = transform.position;
+	}
+
+    void OnMouseDown()
+    {
+        if(isClicked == false)
+        {
+            GameObject theIndicator;
+            if (isCorrect == true)
+            {
+                theIndicator = Instantiate(correctIndicator);
+                MicrogameController.instance.setVictory(true, true);
+                MicrogameController.instance.playSFX(correctSound, volume: 0.5f,
+                panStero: AudioHelper.getAudioPan(0));
+            }
+            else
+            {
+                theIndicator = Instantiate(wrongIndicator);
+                MicrogameController.instance.setVictory(false, true);
+                MicrogameController.instance.playSFX(wrongSound, volume: 0.5f,
+                panStero: AudioHelper.getAudioPan(0));
+            }
+            theIndicator.transform.position = transform.position;
+            rngMaster.GetComponent<KaguyaMemory_RNGDeciderScript>().finished = true;
+            isClicked = true;
+        }
+        
+    }
+
+    // Update is called once per frame
+    void Update () {
```

Comment:
This Update function is not needed, for the same reasons as the one in KaguyaMemory_RNGDeciderScript. Use `Invoke` instead.